plugins/woocommerce/includes/rest-api/Controllers/Version2/class-wc-rest-orders-v2-controller.php

```diff
@@ -210,6 +210,29 @@ protected function get_order_item_data( $item ) {
 
 		// Expand meta_data to include user-friendly values.
 		$formatted_meta_data = $item->get_all_formatted_meta_data( null );
+
+		// Filter out product variations.
+		if ( isset( $product ) && 'true' === $this->request['order_item_display_meta'] ) {
```

Comment:
I wonder if there are any tweaks we might make to this param name? It confused me just a little, because the name doesn't give me a sense of what will be included/what will be excluded. 

That said, I don't really have any great suggestions to offer, either (and maybe it's only unclear to me). If nothing else, though, we probably ought to document the param so it shows up in the docs—and we can clarify what it does there.

Reply:
Definitely open to other ideas. My thinking here is that we're limiting to "display" meta (meta which is meant to be displayed, not private meta and not variation meta) within line items.